### PR TITLE
Rename artifact and package, update POM and README

### PR DESCRIPTION
**Pull request by imagejan**

### CODE REVIEW

**Comment by imagejan** [comment]
I created this _work-in-progress_ PR mainly to test the build on Travis.

On Windows (Git Bash), I get failing tests with this repository:

```
java.io.IOException: Line 3 is not the same length as the first line.
        at net.imagej.table.DefaultTableIOPlugin.open(DefaultTableIOPlugin.java:250)
        at net.imagej.table.DefaultTableIOPlugin.open(DefaultTableIOPlugin.java:59)
        at net.imagej.table.DefaultTableIOPluginTest.openTable(DefaultTableIOPluginTest.java:269)
        at net.imagej.table.DefaultTableIOPluginTest.testParser(DefaultTableIOPluginTest.java:104)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
        at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
        at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
        at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
        at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:379)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:340)
        at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:413)
[ERROR] Tests run: 4, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 0.551 s <<< FAILURE! - in net.imagej.table.DefaultTableIOPluginTest
[ERROR] testSmallTables(net.imagej.table.DefaultTableIOPluginTest)  Time elapsed: 0.047 s  <<< FAILURE!
org.junit.ComparisonFailure:
expected:<Row Header,3.142
[]> but was:<Row Header,3.142
[415926
]>
        at net.imagej.table.DefaultTableIOPluginTest.testSmallTables(DefaultTableIOPluginTest.java:194)

[ERROR] testParser(net.imagej.table.DefaultTableIOPluginTest)  Time elapsed: 0.016 s  <<< FAILURE!
java.lang.AssertionError: Line 3 is not the same length as the first line.
        at net.imagej.table.DefaultTableIOPluginTest.testParser(DefaultTableIOPluginTest.java:110)

[INFO]
[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   DefaultTableIOPluginTest.testParser:110 Line 3 is not the same length as the first line.
[ERROR]   DefaultTableIOPluginTest.testSmallTables:194 expected:<Row Header,3.142
[]> but was:<Row Header,3.142
[415926
]>
[INFO]
[ERROR] Tests run: 8, Failures: 2, Errors: 0, Skipped: 0
```

but the Travis build is fine.

@stelfrich do you have a clue what's causing these failures on Windows?
I'll investigate and push to this PR's branch when I find out.

**Comment by stelfrich** [comment]
> @stelfrich do you have a clue what's causing these failures on Windows?
I'll investigate and push to this PR's branch when I find out.

Just a guess: the tests haven't been designed with Windows in mind, so that line feeds are screwing with them?

**Comment by imagejan** [comment]
> the tests haven't been designed with Windows in mind, so that line feeds are screwing with them?

Yes, Windows wasn't considered obviously, but it's not line feeds but file handles, it seems. The tables of subsequent tests are being written to the same file `table.txt`, but on Windows this file doesn't seem to get deleted correctly between different tests.

At least, this is my current interpretation of what I see happening.

Remaining `table.txt` with all tests enabled:

```text
col1	col2	col3	col4	col5
123	-123.0	+123.0f	0123.0d	0.0
00000	1234567890.0987654321	+NaN	-Infinity	000.000
'should	not,break',unnecessary_quotes,should,break
'some,empty,cells','','',''
```

Remaining `table.txt` after commenting out `testQuote()`:

```text
col1	col2	col3	col4	col5
123.000	-123.000	123.000	123.000	0.000
0.000	1234567890.099	NaN	-Infinity	0.000
0
```

**Comment by imagejan** [comment]
I was able to fix the tests on Windows: there were several places where file handles were created but never closed.

To understand what I changed (mainly adding `try-with-resources`), the "Fix tests" commit is best viewed with ignoring whitespace changes: https://github.com/scijava/scijava-plugins-io-table/pull/3/commits/5cb89a963a87c803a99b91d12c8e6328a93f5090?w=1

File: pom.xml [hunk 1]
```diff
@@ -52,31 +52,36 @@
 			<url>http://imagej.net/User:Leon</url>
 			<properties><id>lnyng</id></properties>
 		</contributor>
+		<contributor>
```
Comment (stelfrich) [inline pom.xml]: I'd suggest adding yourself as `<developer>` and would even suggest handing over more responsibility to you, since I won't have too much time to actively work on this. If not, that's also fine 🙂

Comment (imagejan) [reply pom.xml]: Agreed, I can take over (at least in part) the responsibility for this component. I noticed that, although I'm "owner" in the `scijava` org, I don't have permissions to push on this specific repository, presumably because it was moved with the permissions it had before... Can you make me an admin on this one?

Comment (stelfrich) [reply pom.xml]: Uhm. You should at least have write access to this repository now via your *Hackers* membership. I have, however, managed to screw up the rights on this repository even more by adding the *Hackers* group to it: it seems that @ctrueden is - by some black magic (i.e. reasons that are not known to me) - the only one with admin access to this repository left.

Comment (imagejan) [reply pom.xml]: Right: I don't see the *Settings* tab on this repository, but I do have push permissions and can merge this PR. So I'll go ahead shortly. Thanks!

Comment (ctrueden) [reply pom.xml]: @imagejan Any chance you were using the mobile version of GitHub? I did not see a Settings tab either in that mode. But switching to desktop mode, it was there. In any case, I added you to this repository as a collaborator with admin access.

Comment (imagejan) [reply pom.xml]: I was always using the desktop mode, if I'm not mistaken. But now I also see the Settings tab, so your changes were effective. Thanks a lot, @ctrueden and @stelfrich!

**Comment by stelfrich** [comment]
@imagejan What about changing the POM to something like this:

```
<developers>
	<!-- See https://imagej.net/Team -->
	<developer>
		<id>stelfrich</id>
		<name>Stefan Helfrich</name>
		<url>https://imagej.net/User:stelfrich</url>
		<roles>
			<role>founder</role>
                        <role>debugger</role>
			<role>reviewer</role>
			<role>support</role>
			<role>maintainer</role>
		</roles>
	</developer>
        <developer>
		<id>imagej</id>
		<name>Jan Eglinger</name>
		<url>https://imagej.net/User:Eglinger</url>
		<roles>
			<role>lead</role>
			<role>developer</role>
			<role>debugger</role>
			<role>reviewer</role>
			<role>support</role>
			<role>maintainer</role>
		</roles>
	</developer>
</developers>
```

Once that is done, it looks like we can get this merged and cut a release, which is required for the benchmarks that currently still live in `imagej/imagej-benchmarks` to not have a `SNAPSHOT` dependency. (I actually have the `scijava-table` parts migrated to a new `scijava-benchmarks` repo that I didn't want to push with a `SNAPSHOT` dependency).

**Comment by imagejan** [comment]
Thanks for nudging, @stelfrich, I went ahead, updated the POM according to your suggestion, and released version 0.1.0:
http://maven.imagej.net/#nexus-search;gav~org.scijava~scijava-plugins-io-table~0.1.0 :tada:
